R/zzz.R

```diff
@@ -0,0 +1,9 @@
+# ON LOAD ----
+
+# The functions below define the model information. These access the model
+# environment inside of parsnip so they have to be executed once parsnip has
+# been loaded.
+
+.onLoad <- function(libname, pkgname) {
```

Comment:
What is this file for? It automatically loads the `make_flatline_reg()` function as a `parsnip` model after the `epipredict` package is loaded?

Reply:
+1 I am also curious

Reply:
I am as well... So when the parsnip package is loaded, the code `make_flatline_reg()` is run b/c it needs access some enviro in there. The package stores information about the models in an internal environment object, as it says [here](https://www.tidymodels.org/learn/develop/models/), so we must read and write to that enviro to set and get model-related stuff?

Reply:
Yes, this is right. It took me a while of reading through those docs to get it right.

So, when we write quantile regression, the same stuff applies.